db/migrate/20161125172005_add_braintree_configuration_to_stores.rb

```diff
@@ -1,9 +0,0 @@
-class AddBraintreeConfigurationToStores < SolidusSupport::Migration[4.2]
-  def up
-    Spree::Store.all.each(&:create_braintree_configuration)
-  end
-
-  def down
-    SolidusPaypalBraintree::Configuration.joins(:store).destroy_all
-  end
-end
```

Comment:
Wouldn't it be better to empty the migration instead of removing it, I mean for existing apps that already have embedded it and might get confused otherwise.